packages/@webex/plugin-meetings/src/common/errors/webex-errors.ts

```diff
@@ -145,3 +145,20 @@ class IceGatheringFailed extends WebexMeetingsError {
 
 export {IceGatheringFailed};
 WebExMeetingsErrors[IceGatheringFailed.CODE] = IceGatheringFailed;
+
+/**
+ * @class RetryWithTurnServerFailed
+ * @classdesc Raised whenever we fail to connect media using UDP/TCP and on the retry using TURN server.
+ * @extends WebexMeetingsError
+ * @property {number} code - 30203
+ * @property {string} message - 'Failed to connect media on retry using TURN server'
+ */
+class RetryWithTurnServerFailed extends WebexMeetingsError {
```

Comment:
as discussed we should probably make this a generic add media failure error class

Reply:
Done, changed it to ```AddMediaFailed```